src/components/Navbar.tsx

```diff
@@ -0,0 +1,85 @@
+import { MutableRef } from 'preact/hooks'
+import { useCallback, useMemo, useRef, useState } from 'react'
+import Burger from 'icons/Burger'
+import Button from 'components/Button'
+import Logo from 'components/Logo'
+import NavbarLinks from 'components/NavbarLinks'
+import classnames, {
+  alignItems,
+  backdropBlur,
+  backgroundClip,
+  backgroundColor,
+  display,
+  inset,
+  justifyContent,
+  padding,
+  position,
+  space,
+  zIndex,
+} from 'classnames/tailwind'
+import useBreakpoints from 'hooks/useBreakpoints'
+import useClickOutside from 'hooks/useClickOutside'
+import useThrottle from 'hooks/useThrottle'
+
+const navbar = (visible?: boolean) =>
+  classnames(
+    position('sticky'),
+    inset('top-0'),
+    display('flex'),
+    alignItems('items-center'),
+    justifyContent('justify-between'),
+    padding('py-2', 'px-4', 'md:py-8', 'lg:px-25'),
+    zIndex('z-50'),
+    backgroundClip('bg-clip-padding'),
+    backgroundColor(visible ? 'bg-navbar' : undefined),
+    backdropBlur(visible ? 'backdrop-blur-3xl' : undefined)
+  )
+const buttonsContainer = classnames(
+  display('flex'),
+  alignItems('items-center'),
+  space('space-x-4', 'lg:space-x-10')
+)
+
+export default function () {
+  const { md } = useBreakpoints()
+
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
Good solution, if you want - you can go with pure `tailwind`, like I did for `dosu`: https://github.com/BigWhaleLabs/dosu-frontend/blob/main/src/components/Navbar.tsx

Reply:
Of course, you'll need to change the `transform` and `inset` properties to make the menu be above, not on the right side

Reply:
Let's have both approaches and take a look, which one is better, for dosu or for bwl